src/main/java/org/opentripplanner/middleware/utils/ItineraryUtils.java

```diff
@@ -176,6 +178,37 @@ public static boolean itinerariesMatch(Itinerary referenceItinerary, Itinerary c
         return true;
     }
 
+    /**
+     * Checks that the specified itinerary is on the same day as the specified date/time.
+     * @param itinerary the itinerary to check.
+     * @param dateTime the request date/time to check, in the OTP's time zone.
```

Comment:
Can we rename this to be `requestDateTime`?

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/102/commits/98f91c7d7043e0039a39a1855dc3e40befbf178e.